src/web/index.ts

```diff
@@ -1,6 +1,6 @@
 import CarplayWeb from './CarplayWeb'
 
 export * from '../modules'
-export { WebMicrophone } from './WebMicrophone'
+export { default as WebMicrophone } from './WebMicrophone'
```

Comment:
Output signature of the `web` package doesn't change, but this allows us to stick to default export in `WebMicrophone` instead of named